docs/publishers/config.rst

```diff
@@ -450,6 +450,30 @@ loads.
                               // relative from the receiving iframe.
     }
 
+.. option:: sideBySide
+
+  ``Object``. This option lets you customize how the side-by-side mode behaves.
+
+  For example:
+
+  .. code-block:: javascript
+
+     window.hypothesisConfig = function () {
```

Comment:
We should probably start using modern JS (ie. arrow functions) in our code examples now.

Reply:
Right! I basically copy-pasted another example 😅